README.md

```diff
@@ -44,41 +44,4 @@ Thanks to the following contributors:
 - [NaamloosDT](https://github.com/NaamloosDT)
 - [Kamdzy](https://github.com/Kamdzy)
 - [Bond-009](https://github.com/Bond-009)
-- [POP/STARS](https://github.com/popstars)
-
-## Example
-
-```cs
-//I DO NOT SUGGEST YOU TO WRITE YOUR API KEY, PLEASE PARSE IT FROM AN EXTERNAL FILE.
-private const string API_KEY = "Your_Osu_API_Key"; //You can get one here: https://osu.ppy.sh/p/api
-private IOsuApi Instance { get; }
-
-public MyClass() 
-{
-    Instance = new OsuApi(new OsuSharpConfiguration
-    {
-        ApiKey = API_KEY,
-        ModsSeparator = "|",
-        LogLevel = LoggingLevel.Debug
-    });
-    
-    Instance.Logger.LogMessageReceived += (sender, args) =>
-        args.Logger.Print(args.Level, args.From, args.Message, args.Time);
-    
-    [...]
-}
-
-public async Task GetOsuBeatmapAsync(ulong beatmapId)
-{
-    var beatmap = await Instance.GetBeatmapAsync(beatmapId);
-    Console.WriteLine($"The creator of the map is: {beatmap.Creator}");
-}
-
-public void GetOsuBeatmap(ulong beatmapId)
-{
-    var beatmap = Instance.GetBeatmap(beatmapId);
-    Console.WriteLine($"The creator of the map is: {beatmap.Creator}");
-}
-```
-
-You can also check out the [OsuSharp.Example](https://github.com/Kiritsu/OsuSharp/tree/master/src/OsuSharp.Example) project for a more fully-fledged example.
+- [Kat](https://github.com/thecatitude)
```

Comment:
Should be updated to `[Abyssal](https://github.com/abyssal)` ❤️

Reply:
lmao true... :D